bin/gaia-marionette

```diff
@@ -108,6 +108,12 @@ fi
 # find xpcshell and put it in the path
 XPCSHELL_DIR=$(dirname "$XPCSHELLSDK")
 
+MOCHA_GREP_OPTS=""
+if [ -z "$GAIA_DEVICE_TYPE" ] || [ "$GAIA_DEVICE_TYPE" = "phone" ]; then
+  MOCHA_GREP_OPTS="--invert --grep \[tablet\]|\[tv\]"
+else
+  MOCHA_GREP_OPTS="--grep \[$GAIA_DEVICE_TYPE\]"
+fi
 
```

Comment:
The idea is to keep current tests as is, so those non-tagged tests will run on device type equals to phone. Once we have device type, we only run tests with device tag. So each test will be covered in its related device type, and I don't think there's a general test cases because ui are going to be different.